ad-bank.md

```diff
@@ -0,0 +1,39 @@
+PostHog Ad Bank
+<hr/>
+
+## POSTHOG SELF-HOSTED
+
+* PostHog: product analytics software that’s fast, powerful, and easy to use - all completely self-hosted for maximum SOC 2 & HIPAA compliance. See it in action at posthog.com/product
+* Looking to self-host your own open-source product analytics platform? Look no further than PostHog - a fully-featured platform with no 3rd party cookies. Learn more at posthog.com
+* With PostHog’s self-hosted product analytics platform, your data stays on your infrastructure and you get full access to your production instance. See a demo today at posthog.com 
+* Self-hosted product analytics *and* complete privacy? We’ve got you covered. PostHog’s self-hosted app analytics platform is the perfect solution for privacy-conscious companies like yours. Deploy it free today at posthog.com/signup
+* PostHog’s open-source product analytics can be self-hosted and is great for small teams than want to understand usage in their apps. Get it free today at posthog.com/signup 
```

Comment:
Correction: "Small teams **that** want to"

Reply:
URL doesn't work again.

Reply:
The sentence also reads a bit passive to me. I'd switch it up to something like;

"PostHog's open source analytics platform is great for teams that want to self-host and understand their users. Get it free today at XXX"

Reply:
Amended typo and structure.